utils/installer/lv-config.example.lua

```diff
@@ -44,6 +44,10 @@ O.lang.python.analysis.use_library_code_types = true
 -- javascript
 O.lang.tsserver.linter = nil
 
+-- latex 
+O.vimtex_autosave = false -- saves and compiles on focus change
```

Comment:
Please comment these lines out, to show users that they can change it, but not actually forcing it on all users
because this file is [installed](https://github.com/ChristianChiarulli/LunarVim/blob/rolling/utils/installer/install.sh#L116) for all users